standard/standard_2.php

```diff
@@ -1113,4 +1113,4 @@ function getmypid () {}
  * @since 4.0
  * @since 5.0
  */
-function getmyinode () {}
```

Comment:
Sorry my bad, I've created this PR entirely through github.com. Will use an editor next time :+1: